recipe/meta.yaml

```diff
@@ -19,7 +19,10 @@ source:
 
 build:
   number: {{ build }}
-  script: PETSC_DIR=$PREFIX SLEPC_DIR=$PREFIX $PYTHON -m pip -v install --no-deps .
+  script:
+    - export PETSC_DIR=$PREFIX
```

Comment:
Isn't `SLEPC_DIR` required as well?

Reply:
Yes! Bad copy-paste from the petsc patch.